airbyte-proxy/nginx-auth.conf

```diff
@@ -0,0 +1,45 @@
+events {}
+
+http {
+  server {
+    listen 8000 default_server;
+
+    location / {
+      proxy_set_header Host $host;
+      proxy_set_header X-Real-IP $remote_addr;
+      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+
+      auth_basic "Welcome to Airbyte";
+      auth_basic_user_file /etc/nginx/.htpasswd;
+
+      proxy_pass http://airbyte-webapp:80;
```

Comment:
let's imagine that I want to use a value different from `http://airbyte-webapp:80` e.g. I want to use the proxy in k8s chart (for now I will add the proxy by myself before the chart has it). I guess that this value won't work, see [here](https://github.com/airbytehq/airbyte/blob/master/charts/airbyte-webapp/templates/service.yaml#L5), as the k8s service is called `{{ .Release.Name }}-airbyte-webapp-svc` this means that can be referenced via `http://{{ .Release.Name }}-airbyte-webapp-svc:80.` 

Could we make proxy_pass customizable via an ENV variable??

Reply:
@nicor88 we will tackle Kube related auth in a follow up PR!

Reply:
Thanks, @davinchia, my comment is not necessarily related to k8s (I gave more context about a specific use case).

Adding a flexible PROXY_PASS will make the proxy more flexible, so this comment should enable the groundwork for running easily in k8s. (to add this proxy in k8s we need other few pieces: deployment/service/ingress).